docs/core-api/DAG.md

```diff
@@ -65,23 +65,23 @@ console.log(cid.toString())
 
 A great source of [examples][] can be found in the tests for this API.
 
-## `ipfs.dag.get(cid, [path], [options])`
+## `ipfs.dag.get(cid, [options])`
 
 > Retrieve an IPLD format node
 
 ### Parameters
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| cid | [CID][] | A DAG node that follows one of the supported IPLD formats |
-| path | `String` | An optional path within the DAG to resolve |
+| cid | [CID][] | A CID that resolves to a node to get |
```

Comment:
why do you right the type like that ? for me this `[CID][]` means an optinal array of cids

Reply:
Also a reference link, not an array.